recipes/pymumps/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pymumps" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+  
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/santiagoropb/pymumps/archive/v{{ version }}.tar.gz
```

Comment:
I don't know if this was discussed or not, but using forks in this manner is discouraged. Please use upstream repo https://github.com/PyMumps/pymumps and add any changes needed as patches